contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
> USE_JEMALLOC=OFF

That could be added [here](https://github.com/neovim/neovim/blob/master/CMakeLists.txt#L208).. but USE_BUNDLED_JEMALLOC is for `third-party/CMakeLists.txt` and `USE_JEMALLOC` would be for `CMakeLists.txt`, that's why you need two (or add something to the main Makefile). If you don't mind jemalloc being downloaded, `USE_JEMALLOC=OFF` would be enough.